.github/workflows/validation-not-required.yaml

```diff
@@ -0,0 +1,16 @@
+name: Quarkus Hilla Validation
```

Comment:
Same name a problem?

Reply:
No, on the contrary, it must be the same

Reply:
From the docs 

> Make sure that the name key and required job name in both the workflow files are the same.